resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+                </tr>
+                <tr>
+                    <td>
+                        <p>Fund release not found</p>
+                        <p class="code--block">
+                            lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a>
+                            <br>
+                            accountId: 1000000404
+                        </p>
+                    </td>
+                    <td>
+        <pre class="code--block">
+        {
+          "fundReleaseId": "3456789001",
+          "eventDate": "2017-05-10",
+          "eventType": "Extension one"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">404 (Not Found)</code></p>
+        <pre class="code--block">
+        {
+          "code": "FUND_RELEASE_NOT_FOUND",
+          "message": "The fundReleaseId does not match HMRC’s records"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <p>Extension already exists</p>
+                        <p class="code--block">
+                            lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a>
+                            <br>
+                            accountId: 0000000409
+                        </p>
+                    </td>
+                    <td>
+        <pre class="code--block">
+        {
+          "fundReleaseId": "3456789001",
+          "eventDate": "2017-05-10",
+          "eventType": "Extension one"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">409 (Conflict)</code></p>
+        <pre class="code--block">
+        {
+          "code": "LIFE_EVENT_ALREADY_EXISTS",
+          "message": "The investor’s life event has already been reported",
+          "lifeEventId": "6789000001"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <p>The associated fund release has been superseded</p>
+                        <p class="code--block">
+                            lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a>
+                            <br>
+                            accountId: 2000000409
+                        </p>
+                    </td>
+                    <td>
+        <pre class="code--block">
+        {
+          "fundReleaseId": "3456789000",
+          "eventDate": "2017-05-10",
+          "eventType": "Extension one"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">409 (Conflict)</code></p>
+        <pre class="code--block">
+        {
+          "code": "FUND_RELEASE_SUPERSEDED",
+          "message": "This fund release has already been superseded"
```

Comment:
Missing a comma at the end here, please add. (Was missing in the RAML)

Reply:
Good spot 👍